.github/workflows/release.yaml

```
@@ -18,50 +18,31 @@ concurrency:
 jobs:
   build-library:
     name: Library ${{ matrix.architecture }}
```

actionlint (1.7.12)

[error] property "architecture" is not defined in object type {}

(expression)